resources/postman_collection.md

```diff
@@ -0,0 +1,38 @@
+# Postman Collection for MeiliSearch
+
+Are you tired to use the `curl` or `jq` command in your terminal to test MeiliSearch?
```

Comment:
Since you do not use `jq` to make requests to meiliSearch, I don't think it should be added here. It is only used to pretty show the results and other libraries other than `jq` are out there to do the same job. It is not essential information. Only mentioning curl works for everyone.

Also I added a little flavor text to introduce. 
```suggestion
Are you tired to use the `curl` command in your terminal to test MeiliSearch? It can be tedious to re-write every route when wanting to try out an API. 
```